infra/scoped/api-integration.tf

```diff
@@ -70,6 +70,35 @@ resource "aws_api_gateway_integration" "users_userid_password_put" {
   }
 }
 
+# [OPTIONS] /users/:user_id/registration
+
+resource "aws_api_gateway_integration" "users_userid_registration_options" {
+  rest_api_id = aws_api_gateway_rest_api.identity.id
+  resource_id = aws_api_gateway_resource.users_userid_registration.id
+  http_method = aws_api_gateway_method.users_userid_registration_options.http_method
+
+  integration_http_method = "POST"
```

Comment:
Why is this POST when the comment says OPTIONS?

Reply:
Good question – this was broadly copy/paste from the [`users/:user_id/password` block above](https://github.com/wellcomecollection/identity/pull/296/files/6312d9d7d63ab39b12738f7405e263f47679da19#diff-f13e44ae1ed19905dd828a72ee53102577e546d82a2dcb1ec87600dd2bc67d7aR45-R71), where it is also `POST` under the same circumstances. Do both of these need updating?

Reply:
Great. Thanks for clarifying